server/dba/seeders/prod/05-forestLocations.js

```diff
@@ -3,51 +3,439 @@
+        district: null,
+        allowed: false,
+        type: 'area',
+        description: 'Campgrounds or administrative sites',
+        image_filename: null
+      },
+      {
+        id: 5,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'area',
+        description: '300 feet of streams or lakes',
+        image_filename: null
+      },
+      {
+        id: 6,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'area',
+        description: 'Areas posted "No Christmas Tree Cutting”',
+        image_filename: null
+      },
+      {
+        id: 7,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'area',
+        description: 'East of Little Crater Lake Campground on Road 58, because of Warms Springs Indian Reservation',
+        image_filename: null
+      },
+      {
+        id: 8,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'area',
+        description: 'Do not remove Pacific Yew trees',
+        image_filename: null
+      },
+      {
+        id: 9,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'area',
+        description: 'Campgrounds, administrative sites, or summer home/rec. residence areas',
+        image_filename: null
+      },
+      {
+        id: 10,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Highway 26',
+        image_filename: null
+      },
+      {
+        id: 11,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Highway 224',
+        image_filename: null
+      },
+      {
+        id: 12,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Road 42 (Timothy Lake Road)',
+        image_filename: null
+      },
+      {
+        id: 13,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Road 50 (Timberline Road)',
+        image_filename: null
+      },
+      {
+        id: 14,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Road 2645 (Westleg Road)',
+        image_filename: null
+      },
+      {
+        id: 15,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Road 2618 (Salmon River Road)',
+        image_filename: null
+      },
+      {
+        id: 16,
+        forest_id: 3,
+        district: null,
+        allowed: false,
+        type: 'road',
+        description: 'Road 2627, 2627-207',
+        image_filename: null
+      },
+      {
+        id: 17,
+        forest_id: 3,
+        district: 'Clackamas',
+        allowed: true,
+        type: 'road',
+        description: '4210',
```

Comment:
As per my prior PR, please make the road display consistent in the seeders.

Reply:
@lauraGgit this is being updated as a task in https://github.com/nciinc/fs-intake-module/issues/93